UNL-CSE-AMBASSADORS/VITA
# Remove DependentClient from system

Removed the DependentClient table from the system.

**CHANGES TO PROD DB**
After this is deployed, the following script will need to be run against the prod database:
```
TRUNCATE TABLE DependentClient;
DROP TABLE DependentClient;
```

**TESTING**
* Go to the appointment signup page.
  * The Add Dependents area should no longer be there
  * Appointment signup should still function properly
* Go to the queue
  * None of the appointments should have dependents on them when you click on one
  * The queue should still function properly
* Go to Print Documents
  * Download an appointment schedule for any day
    * It should no longer have a "Number of Returns" column
    * It should still function properly
